networkdb/cluster.go

```diff
@@ -479,26 +479,34 @@ func (nDB *NetworkDB) bulkSyncTables() {
 }
 
 func (nDB *NetworkDB) bulkSync(nodes []string, all bool) ([]string, error) {
+	syncNodes := nodes
+	retried := false
+retry:
 	if !all {
 		// If not all, then just pick one.
-		nodes = nDB.mRandomNodes(1, nodes)
+		syncNodes = nDB.mRandomNodes(1, nodes)
 	}
 
-	if len(nodes) == 0 {
+	if len(syncNodes) == 0 {
 		return nil, nil
 	}
 
-	logrus.Debugf("%s: Initiating bulk sync with nodes %v", nDB.config.NodeName, nodes)
+	logrus.Debugf("%s: Initiating bulk sync with nodes %v", nDB.config.NodeName, syncNodes)
 	var err error
 	var networks []string
-	for _, node := range nodes {
+	for _, node := range syncNodes {
 		if node == nDB.config.NodeName {
 			continue
 		}
 
 		networks = nDB.findCommonNetworks(node)
 		err = nDB.bulkSyncNode(networks, node, true)
 		if err != nil {
+			if !all && !retried {
+				logrus.Errorf("bulk sync attempt to node %s failed: %v", node, err)
+				retried = true
+				goto retry
+			}
```

Comment:
Imagining analyzing these logs afterwards, don't you guys think it would make sense to add a 

```
else if retried {
   logrus.Debugf/Infof("bulk sync now succeeded to node %s", node)
}
```

in case `err ==nil`.